server/src/test/java/org/elasticsearch/index/IndexSettingsTests.java

```diff
@@ -344,14 +344,18 @@ public void testStatelessDefaultRefreshInterval() {
     }
 
     public void testStatelessFastRefreshDefaultRefreshInterval() {
-        IndexMetadata metadata = newIndexMeta(
-            "index",
-            Settings.builder()
-                .put(IndexMetadata.SETTING_VERSION_CREATED, Version.CURRENT)
-                .put(EXISTING_SHARDS_ALLOCATOR_SETTING.getKey(), "stateless")
-                .put(INDEX_FAST_REFRESH_SETTING.getKey(), true)
-                .build()
-        );
+        IndexMetadata metadata = IndexMetadata.builder("index")
+            .system(true)
+            .settings(
+                indexSettings(Version.CURRENT, 1, 1).put(
+                    Settings.builder()
+                        .put(IndexMetadata.SETTING_VERSION_CREATED, Version.CURRENT)
+                        .put(EXISTING_SHARDS_ALLOCATOR_SETTING.getKey(), "stateless")
+                        .put(INDEX_FAST_REFRESH_SETTING.getKey(), true)
+                        .build()
+                )
```

Comment:
I think this could be simplified to:

```suggestion
                indexSettings(Version.CURRENT, 1, 1)
                        .put(IndexMetadata.SETTING_VERSION_CREATED, Version.CURRENT)
                        .put(EXISTING_SHARDS_ALLOCATOR_SETTING.getKey(), "stateless")
                        .put(INDEX_FAST_REFRESH_SETTING.getKey(), true)
```